volatility3/framework/plugins/linux/hidden_modules.py

```diff
@@ -29,46 +30,30 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
             requirements.PluginRequirement(
                 name="lsmod", plugin=lsmod.Lsmod, version=(2, 0, 0)
             ),
+            requirements.VersionRequirement(
+                name="linux_utilities_modules",
+                component=linux_utilities_modules.Modules,
+                version=(2, 0, 0),
+            ),
         ]
 
-    @classmethod
+    @staticmethod
+    @Deprecation.deprecated_method(
```

Comment:
Yep, shouldn't start with a capital D, definitely prefer these methods moving to a module of their own please.

Reply:
Happy for you to do it in this PR, but fine if you just want to put a bug number here and we can do it later, but we might as well do it before use becomes widespread.